app/src/test/java/org/p2p/wallet/swap/koin/CheckModulesTest.kt

```diff
@@ -99,6 +104,14 @@ class CheckModulesTest : KoinTest {
         every { getExternalFilesDir(any()) }.returns(externalDirMock)
     }
 
+    private val environmentManagerMock: EnvironmentManager = mockk {
```

Comment:
sorry, but it's wrong
you just need to mock shared prefs, no need to create mock for our own class